### DISCUSSION TITLE
What is the origin of ZLUDA's name?

### DISCUSSION THREAD

**Discussion by BaalNetbek** [vosen/ZLUDA#192] [Q&A] [answered]
What is the origin of ZLUDA's name? 
Does it refer to Polish 'złuda' (delusion)? Which could mean that CUDA's incompatibilty with non nVidia hardware was just an delusion.

**Comment by vosen** [comment, marked as answer]
Yes, the reference is to złuda. Though translating it to more neutral "mirage"/"illusion" would be more accurate.

As for the origin, the day before the first public release the project's name was still "notCUDA" (if you browse old git revisions you can see it all over the place). I had to come with a less dodgy name and since CUDA already means something in Polish ("miracles"/"wonders") I decided on a reverse route and picking a similarly sounding Polish word. There's no hidden message here.

I suppose after adding the AMD GPU support the project should be renamed to RUDA
